cross/python311/Makefile

```diff
@@ -149,6 +149,7 @@ CROSSENV_WHEELS += poetry==1.6.1
 CROSSENV_WHEELS += scikit-build==0.17.6
 CROSSENV_WHEELS += setuptools-rust==1.7.0
 CROSSENV_WHEELS += setuptools-scm==7.1.0
+CROSSENV_WHEELS += expandvars==0.12.0
```

Comment:
this will be reverted and migrated to `ADDITIONAL_CROSSENV_WHEELS` when #6040 is merged.